test/e2e/config/operator.yaml

```diff
@@ -27,7 +27,7 @@ intervals:
 
 variables:
   RANCHER_VERSION: "v2.8.1"
-  KUBERNETES_VERSION: "v1.27.0"
+  KUBERNETES_VERSION: "v1.27.3"
```

Comment:
how about using newer version here? kind image was already v1.28.0 prior to this change